playground/src/views/Editor.vue

```diff
@@ -91,6 +100,12 @@ class Weblet {
 
 @Component({
   name: "Editor",
+  data: function () {
+    return {
+      envs: { dev: "DEVNET", test: "TESTNET", qa: "QANET", main: "MAINNET" },
+      network: process.env.NETWORK != "" ? process.env.NETWORK : "DEVNet",
```

Comment:
I think we can change `DEVNet` to `dev`
Also I think we always have `dev` as a default network

Reply:
Th default network is already set as dev in line 106.